cypress/support/guillotina.js

```diff
@@ -19,14 +19,15 @@ export function setupGuillotina() {
     body: { id: 'cms' },
   }).then(response => console.log('cms add-on installed'));
 
-  cy.request({
-    method: 'POST',
-    url: `${api_url}/container/@addons`,
-    headers,
-    body: { id: 'dbusers' },
-  }).then(response => console.log('dbusers add-on installed'));
-
   // guillotina_dbusers is no longer installed in the guillotina_cms docker image
+  //
```

Comment:
@ksuess can you elaborate on why we don't need this any longer. Just wondering...

Reply:
unrelated to this PR: reverted

Reply:
Your revert brought back the commted out code block.